<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MappingWindow</class>
 <widget class="QMainWindow" name="MappingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="mapping">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>130</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Mapping </string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_table_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>View table </string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_table_1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>View table </string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbar_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>272</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="compare">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>130</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Compare </string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbar_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>392</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="view_table_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>400</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>View table </string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbar_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>332</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="preprocess">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>130</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Preprocess</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_table">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Get table</string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_table_name">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>20</y>
      <width>138</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>View table name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_table_0">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>View table </string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>270</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_preprocess">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>130</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_compare">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>130</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_mapping">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>130</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_1">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>280</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>340</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_0">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>400</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
